Tilers/CityTiler/citym_building.py

```diff
@@ -72,7 +72,21 @@ def sql_query_objects(buildings):
         return query
 
     @staticmethod
-    def sql_query_geometries(offset, buildings_ids_arg):
+    def sql_query_textures(image_uri):
+        """
+        :param buildings: a list of CityMBuilding type object that should be sought
+                        in the database. When this list is empty all the objects
+                        encountered in the database are returned.
+
+        :return: a string containing the right SQL query that should be executed.
+        """
+
+        query = \
+            "SELECT tex_image_data FROM tex_image WHERE tex_image_uri = '" + image_uri + "' "
+        return query
+
+    @staticmethod
+    def sql_query_geometries_with_texture_coordinates(offset, buildings):
```

Comment:
Place here some design notes indicating why textures coordinates must
be retrieved in a single querry with the geometries. 
This design choice is to be opposed with first calling sql_query_geometries()
and then calling a newly created sql_query_texture_coordinates(concerned_geometries).